app/param_filters.go

```diff
@@ -74,5 +74,10 @@ func consensusParamFilter(msg sdk.Msg) error {
 		return errors.Wrapf(sdkerrors.ErrUnauthorized, "invalid validator parameters")
 	}
 
+	evidenceParams := coretypes.DefaultConsensusParams().ToProto().Evidence
+	if !updateParams.Evidence.Equal(evidenceParams) {
+		return errors.Wrapf(sdkerrors.ErrUnauthorized, "invalid evidence parameters")
+	}
```

Comment:
this appears to be the thing breaking the test, since I don't think the existing params are the default maybe?

we're attempting to set the evidence params to what they are onchain in the squareSize test, but that's not matching the default, therefore that's not allowing any param changes